src/components/navs/AppNav/AppSidebar/SidebarContent.vue

```diff
@@ -120,7 +120,7 @@ watch(blockNumber, async () => {
     <div
       class="flex flex-col justify-center px-4 h-20 border-b border-gray-800"
     >
-      <AppLogo forceDark />
```

Comment:
This causes problems, why did it change?
- Open the preview in light mode
- Make the window narrow enough so the sidebar is togglable
- Open the sidebar, you should see that the logo is the wrong color.
<img width="789" alt="Screenshot 2023-03-01 at 10 59 28" src="https://user-images.githubusercontent.com/2406506/222120728-25c4e3ed-9c1f-4564-8bd0-95675866658d.png">